typing_extensions/src_py3/typing_extensions.py

```diff
@@ -155,27 +107,16 @@ def _check_methods_in_mro(C, *methods):
     'TYPE_CHECKING',
 ]
 
-# Annotated relies on substitution trees of pep 560. It will not work for
-# versions of typing older than 3.5.3
-HAVE_ANNOTATED = PEP_560 or SUBS_TREE
-
 if PEP_560:
     __all__.extend(["get_args", "get_origin", "get_type_hints"])
 
-if HAVE_ANNOTATED:
-    __all__.append("Annotated")
-
-# Protocols are hard to backport to the original version of typing 3.5.0
-HAVE_PROTOCOLS = sys.version_info[:3] != (3, 5, 0)
-
-if HAVE_PROTOCOLS:
-    __all__.extend(['Protocol', 'runtime', 'runtime_checkable'])
+__all__.extend(['Annotated', 'Protocol', 'runtime', 'runtime_checkable'])
```

Comment:
Given these names are now unconditionally added to `__all__`, I think they should be in the initial definition -- but not sure in which category to put them

A